docs/static/persistent-queues.asciidoc

```diff
@@ -29,6 +29,15 @@ messages stored in the persistent queue until delivery succeeds at least once.
 NOTE: You must set `queue.checkpoint.writes: 1` explicitly to guarantee
 maximum durability for all input events. See <<durability-persistent-queues>>.
 
++
+NOTE: The Persistent Queue is under active development and continues to improve
+with each release of Logstash; using the latest available version of Logstash
+will ensure you're using the best available version of this and other features.
+To enable the smoothest transition when upgrading, especially from versions
+prior to 6.3.0, you're encouraged to
```

Comment:
I think it's going to be very weird to mention 6.3.0 in releases that occurred prior to 6.3.0. I would remove the mention of 6.3.0 from the note all together and make sure the note shows up in all relevant versions (where the queue.drain option exists) up to, but not including, the version where draining the queue is required.